src/ConnectedMode.UnitTests/UI/ManageConnections/ManageConnectionsViewModelTest.cs

```diff
@@ -224,4 +265,9 @@ private void MockTryGetConnections(List<Connection> connections)
             return true;
         });
     }
+
+    private static Connection CreateSonarCloudConnection()
+    {
+        return new Connection(new ConnectionInfo("https://sonarcloud.io/mySecondOrg", ConnectionServerType.SonarCloud), false);
```

Comment:
The `Id` for SonarCloud is just the organization name.

```suggestion
        return new Connection(new ConnectionInfo("mySecondOrg", ConnectionServerType.SonarCloud), false);
```

Reply:
@vnaskos-sonar I am not sure about this. I have talked with @georgii-borovinskikh-sonarsource and initially he disagreed with this approach, but later last week he seemed to agree. 

I will not change the implementation, but instead update the task: https://sonarsource.atlassian.net/browse/SLVS-1419. When implementing the task, maybe we can have the discussion again.

Reply:
Actually, I changed it as suggested. Because the code sets the OrganizationKey for the connection id for now, so the tests should reflect that

Reply:
This is the only place where the Id for sonarcloud is a URL. The production code is using just the organization. 
```
    {
      "id": "duncanp-sonar-test",
      "setting": {
        "IsSmartNotificationsEnabled": true
      },
      "organizationKey": "duncanp-sonar-test",
      "serverUri": null
    },
```

If we keep this here, we will introduce an inconsistency that will confuse things later.